app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsContributionsFragment.kt

```diff
@@ -370,7 +370,8 @@ class SuggestedEditsContributionsFragment : Fragment(), SuggestedEditsContributi
             view.contribution = contribution
             view.setTitle(contribution.description)
             view.setDiffCountText(contribution)
-            view.setDescription(contribution.title)
+            view.setDescription(if (contribution.editType == EDIT_TYPE_IMAGE_TAG || contribution.editType == EDIT_TYPE_IMAGE_CAPTION)
+                contribution.title.replaceFirst("File:", "") else contribution.title)
```

Comment:
I think we can just remove the namespace without an extra logic.

Also, you can use `StringUtil.removeNamespace(contribution.title)` to remove the namespace.

Reply:
I noticed that the `File:` should be removed in the detail page as well.

Reply:
Correct @cooltey done!